doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
 
-## $PlatformShort$ Simple Bar Chart
+There are several common use cases for choosing a Bar Chart: 
 
-<!-- TODO copy and combine content (code snippets, description) from these topics:
-    data-chart-type-category-bar-series.md
--->
+- You need to show trends over time or a numeric value change in a category of data 
+- You need to compare data values of 1 or more data series 
+- You want to show a part-to-whole comparison 
+- You want to show top or bottom percentage of categories 
+- Analyzing multiple data points grouped in sub-categories (Stacked Bar) 
+
+The $PlatformShort$ Bar Chart includes several variants based on your data or how you want to tell the correct story with your data. These include: 
+
+- Grouped Bar Chart 
+- Stacked Bar Chart 
+- Polar Bar Chart 
+- Stacked 100 Bar Chart 
+
+Bar Chart Best Practices: 
+
+- Start you numeric Axis at 0 
+- Use a single color for the bars  
+- Be sure the space separating each bar is 1/2 the width of the bar itself 
+- Be sure ranking or comparing ordered categories (items) are  sorted in increasing or decreasing order 
+- Right-align category values on the Y-Axis (left side labels of chart) for readability 
+
+Don't Use a Bar Chart When: 
+
+- You have too much data so the Y-Axis can't fit in the space or is not legible 
+- You need a Detailed Time-Series analysis  - consider a Line Chart with a Time-Series for this type of data.
+
+Bar Chart Data Structure: 
+
+- The data source must be an array or a list of data items 
+- The data source must contain at least one data item  
+- The list must contain at least one data column (string or date time)  
+- The list must contain at least one numeric data column 
+
+Common $PlatformShort$ Bar Chart Scenarios: 
+
+- Sales Management 
+- Inventory Management 
+- Stock Charts 
+- Any String Value Comparing a Numeric Value or Time-Series Value 
+
+## $PlatformShort$ Bar Chart Example
+
+In this example, the bar chart is comparing the total number of Olympic Medals (the Category) won between the USA and Russia from 1996 to 2016. The Y-Axis, or labels on the left of the chart, are displaying the time-series values of Year. The X-Axis, or labels on the bottom of the chart, are displaying the number of medals won.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=28c84be1-a43f-4c5f-8b18-5018755bf116&mode=build

Reply:
**NEW EXAMPLE TO BE USED**

https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=c2eae4b8-1622-41e2-b706-867a4fd3a10f&mode=design